core/templates/dev/head/header_css_libs.html

```diff
@@ -26,4 +7,6 @@
 <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet">
 <link rel="stylesheet" type="text/css" media="screen"
-      href="/third_party/static/font-awesome-4.4.0/css/font-awesome.min.css">
+      href="/css/oppia.css">
+<link rel="stylesheet" type="text/css" media="screen"
```

Comment:
I'm not super-sure about this. Can you talk me through the story of a developer who wants to add and test out a new CSS library? What are the steps they should follow?

In general, I feel it would be better to have non-minified CSS (no build step) on dev and minified CSS on prod. Note scripts/build.py -- that is the script that converts the dev files to prod files. Note also that it is possible for the frontend to know whether it's in dev or prod mode -- this information is passed through the Jinja template using DEV_MODE.

Overall, I think we need to find a way such that individual CSS files can be added to a list, there is only one location (source of truth) for that list, and the files are unminified in dev and minified in prod.